utilities/update_links.py

```diff
@@ -1,24 +1,34 @@
 import os
 import pandas as pd
 pd.option_context('display.max_rows', None,
-                       'display.max_columns', None,
-                       'display.precision', 3)
+                  'display.max_columns', None,
+                  'display.precision', 3)
 
 filepath = r"..\ons-spark\ons-spark"
-old_string = ""
-new_string = ""
+old_string = "sprak"
+new_string = "spark"
 extensions = ('.md', '.ipynb')
```

Comment:
lines 7-10 should either be removed or added into the documentation for the wrapper function as an example usage

Reply:
added under Example